### PR TITLE
add bépo_new, an adaptation of bepo for mobiles

### PR DESCRIPTION
**Pull request by MXC48**
#2494 
it's my first contribution, tell me if I'm making a mistake

### CODE REVIEW

**Comment by MXC48** [comment]
Anyone can review ?

File: app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/bepo_new.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    { "$": "auto_text_key", "code":   97, "label": "a" },
+    { "$": "auto_text_key", "code":  117, "label": "u" },
+    { "$": "auto_text_key", "code":  105, "label": "i" },
+    { "$": "auto_text_key", "code":  101, "label": "e" },
+    { "$": "auto_text_key", "code":   99, "label": "c" },
+    { "$": "auto_text_key", "code":  116, "label": "t" },
+    { "$": "auto_text_key", "code":  115, "label": "s" },
+    { "$": "auto_text_key", "code":  114, "label": "r" },
+    { "$": "auto_text_key", "code":  110, "label": "n" },
+    { "$": "auto_text_key", "code":  109, "label": "m" },
```
Comment (patrickgold) [inline app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/bepo_new.json:24]:
```suggestion
    { "$": "auto_text_key", "code":  109, "label": "m" }
```

There is an additional comma which shouldn't be here

**Review by patrickgold — CHANGES_REQUESTED** [review]
Thanks for your contribution! There is a minor issue with the JSON syntax, else it looks good.

**Comment by patrickgold** [comment]
Hi there!

We're closing this PR because the v0.5 development cycle is about to begin, and it will bring significant changes to the layout code. This would likely cause merge conflicts with your PR. Additionally, custom keyboard layouts will mostly be moved to the addons store with v0.5. We hope you understand and thank you for your contribution!

**Pull request closed without merging**